cdap-docs/reference-manual/source/java-client-api.rst

```diff
@@ -118,6 +120,34 @@ ApplicationClient
   // List programs belonging to an application
   appClient.listPrograms("Purchase");
 
+.. _preferences-client:
+PreferencesClient
+-----------------
+::
+  ClientConfig clientConfig;
+
+  //Construct the client used to interact with CDAP
+  PreferencesClient preferencesClient = new PreferencesClient(clientConfig);
+
+  Map<String, String> propMap = Maps.newHashMap();
+  propMap.put("k1", "v1");
+  //Set Preferences at Instance level
+  preferencesClient.setInstancePreferences(propMap);
+
+  //Get Preferences at Instance level
+  preferencesClient.getInstancePreferences();
+
+  //Delete Preferences at Instance level
+  preferencesClient.deleteInstancePreferences();
+
+  //Set Preferences of MyApp application which is deployed in Dev namespace
```

Comment:
// Set Preferences of the MyApp application which is deployed in the Dev namespace